src/common/models/Artifacts.ts

```diff
@@ -0,0 +1,4 @@
+export type Artifacts = {
```

Comment:
It is usually better to name models in singular nouns, so _Artifact_ instead of _Artifacts_. 
This is a convention that was followed in the project up until now, not a very big deal tho.

Reply:
updated